### PR TITLE
Clarify IP detect

### PR DESCRIPTION
**Pull request by joel-hamill**
## Description

https://jira.mesosphere.com/browse/DOCS-1538
- https://github.com/mesosphere/dcos-docs-enterprise/pull/1259

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Review by branden — APPROVED** [review]

File: 1.9/installing/custom/advanced.md [hunk 1]
```diff
@@ -91,13 +91,16 @@ Your cluster must meet the software and hardware [requirements](/docs/1.9/instal
     - '.baz.com'
     ```
 
-2. Create a `ip-detect` script
+2.  Create a `ip-detect` script.
```
Comment (sascala) [inline 1.9/installing/custom/advanced.md]: s/a/an

File: 1.9/installing/custom/advanced.md [hunk 2]
```diff
@@ -91,13 +91,16 @@ Your cluster must meet the software and hardware [requirements](/docs/1.9/instal
...
+    **Important:** 
+    
+    - The IP address of a node must not change after DC/OS is installed on the node. For example, the IP address should not change when a node is rebooted or if the DHCP lease is renewed. If the IP address of a node does change, the node must be [wiped and reinstalled](/docs/1.9/installing/custom/uninstall/).
+    - The script must return the same IP address as specified in the `config.yaml`. For example, if the private master IP is specified as `10.2.30.4` in the `config.yaml`, your script should return this same value when run on the master. 
 
-    Create an IP detection script for your environment and save as `genconf/ip-detect`. This script needs to be `UTF-8` encoded and have a valid [shebang](https://en.wikipedia.org/wiki/Shebang_%28Unix%29) line. You can use the examples below.
+    Create an IP detection script for your environment and save as `genconf/ip-detect`. This script must be `UTF-8` encoded and have a valid [shebang](https://en.wikipedia.org/wiki/Shebang_%28Unix%29) line. You can use the examples below.
```
Comment (sascala) [inline 1.9/installing/custom/advanced.md]: elsewhere it's called an "IP detect script".

File: 1.9/installing/custom/cli.md [hunk 3]
```diff
@@ -60,9 +60,12 @@ Your cluster must meet the software and hardware [requirements](/docs/1.9/instal
...
 
-    **Important:** The IP address of a node must not change after DC/OS is installed on the node. For example, the IP address must not change when a node is rebooted or if the DHCP lease is renewed. If the IP address of a node does change, the node must be [wiped and reinstalled][7].
+    **Important:** 
+    
+    - The IP address of a node must not change after DC/OS is installed on the node. For example, the IP address should not change when a node is rebooted or if the DHCP lease is renewed. If the IP address of a node does change, the node must be [wiped and reinstalled](/docs/1.9/installing/custom/uninstall/).
+    - The script must return the same IP address as specified in the `config.yaml`. For example, if the private master IP is specified as `10.2.30.4` in the `config.yaml`, your script should return this same value when run on the master. 
 
     Create an IP detection script for your environment and save as `genconf/ip-detect`. This script needs to be `UTF-8` encoded and have a valid [shebang](https://en.wikipedia.org/wiki/Shebang_%28Unix%29) line. You can use the examples below.
```
Comment (sascala) [inline 1.9/installing/custom/cli.md]: same comment as above re "IP detection".

**Comment by joel-hamill** [comment]
@sascala fixed

**Pull request merged**